### ISSUE TITLE
[Profil et prédiction d'occupation de stations de vélos libre-service] Damien Garaud

### ISSUE THREAD

**Issue by garaud** [TlseDataScience/meetup-cfp#8]
## Le speaker
Damien Garaud, Data Scientist & Développeur chez [Oslandia](http://oslandia.com)

* Twitter handle: [@jazzydag](https://twitter.com/jazzydag)

## Sujet de votre conférence
Profil et prédiction d'occupation de stations de vélos libre-service

## Description de votre conférence

Cette présentation est un cas pratique d'exploration et de traitement de données dont l'objectif est de prédire l'occupation des stations de vélos libre-service. Grâce aux Données Ouvertes aka *Open Data* de certaines métropoles comme Lyon ou Bordeaux, il est possible de récupérer des données sur l'occupation des vélos libre-service. La présentation suivra un cheminement assez logique quand on travaille avec de la donnée :

1. récupération et formattage
2. exploration et visualisation
3. traitement et prédiction

Après une brève introduction à une bibliothèque Python de *data pipeline*, la deuxième partie montrera un cas concret de profil horaire avec l'utilisation d'un modèle de *Machine Learning* dit non-supervisé (`KMeans`). Puis nous nous concentrerons sur la préparation de la donnée appelée aussi *feature engineering* afin que le modèle de prédiction, ici `XGBoost`, puisse prévoir au mieux l'occupation de chaque station sur 1 heure par exemple.

Vous pourrez découvrir ou reconnaitre certaines bibliothèques [Python](https://www.python.org/) qui ont été utilisées dans ce projet : [luigi](http://luigi.readthedocs.io/), [pandas](http://pandas.pydata.org/), [seaborn](http://seaborn.pydata.org/), [scikit-learn](http://scikit-learn.org), [folium](https://github.com/python-visualization/folium) ou [xgboost](http://xgboost.readthedocs.io/en/latest/).

## Informations diverses
* Thématique, Labels : data science, prediction, machine learning, python, open data, SIG
* Niveau de difficulté (débutant|avancé|confirmé) : débutant
* Durée (quickie 15-30min, conférence >1h) : 20-30min
* Format (slides, live-coding, les deux): slides et jupyter notebook
* Dispo ou indispo (après mars, dès que possible, pas entre X et Y): décembre / février

**Comment by garaud** [comment]
* Une remarque : j'ai mis "débutant" parce que je vais tenter de vulgariser et il y aura quelques lignes de code Python. On pourra changer si vous pensez qu'à la lecture de la description, ça penche plus vers de "l'avancé".

* Et une question : pour des questions pratiques et comme je viendrai de Bordeaux, je voulais en savoir plus sur le lieu, comment s'y rendre depuis la Gare, l'heure de fin etc. Merci !
